### ISSUE TITLE
Items for 1.9 release

### ISSUE THREAD

**Issue by RubenWillems** [ccnet/CruiseControl.NET#275]
There are 2 new project boards, so we can easily follow up what needs to be done.
Comment on this thread what items need to go on which board.

**Comment by mintsoft** [comment]
I'd love 2.0 to have dynamic adding and reloading of projects without it having to stop all projects.

**Comment by savornicesei** [comment]
@RubenWillems
So everything that's on release/CCNet-1.8 branch was also added to master, right? 
Is it possible to tag the commit from which 1.8.5 version was released? It would ease regression testing and release notes for the 1.9 version.

**Comment by RubenWillems** [comment]
Hi

everything should be in the master branch also.
you want an extra tag in the master branch where the 1.8.5 was branched
from?


with kind regards
Ruben Willems

**Comment by savornicesei** [comment]
Hi,
Yes, if it's possible.

Best,
Simo

**Comment by RubenWillems** [comment]
Hi

My git knowledge is bad, so bare with me.
it does not seem possible, but from looking around, the only safe way is to
use
git reflog show --no-abbrev <branch name>

and those logs are automatically deleted after 90 days.

I tried some other commands, but the output was not convincing,
so unless somebody can provide me with the commit hash, it is not possible,
sorry.

There are also changes merged back, making it even hards to do.
When we move to GIT, it was very new to all of us, and we did not tag it
when we made the branch :-(


source :
https://stackoverflow.com/questions/3998883/git-how-to-find-commit-hash-where-branch-originated-from



with kind regards
Ruben Willems

**Comment by savornicesei** [comment]
Yeah, kind of expected. I could not get some clear history from git either. Does the svn repo still exist on sourceforge?

**Comment by RubenWillems** [comment]
Hi

I doubt that, all links where replaced to point to github.

with kind regards
Ruben Willems
